### PR TITLE
Grammatical tweaks

### PR DESCRIPTION
**Pull request by machawk1**

### CODE REVIEW

**Review by dcwalk — APPROVED** [review]

File: web_archiving/readme.md [hunk 1]
```diff
@@ -49,7 +49,7 @@ Capabilities are evaluated as "out of the box" functionality. In cases where a s
 
 * _**WUI**:_ Whether the software offers a web-based user interface.
 
-* _**Library API**:_ Does it provide an explicit libary interface? (I.e., does it provide an API so that you can use it in your own program?)
+* _**Library API**:_ Does it provide an explicit libary interface? (i.e., does it provide an API so that you can use it in your own program?)
```
Comment (mhucka) [inline web_archiving/readme.md:52]: This proposed edit is actually incorrect: the parenthesized sentence is a full sentence, and therefore, should begin with a capital letter. 

Here are some references to back that up (admittedly not authoritative, but I don't think it's worth spending a lot of time on this):

- https://english.stackexchange.com/a/37204
- https://ell.stackexchange.com/questions/56861/capitalize-i-e-and-e-g

(We can debate whether it's good style to begin a sentence with "i.e." or "e.g.", and in fact, I generally avoid it in more formal writing.)

File: web_archiving/readme.md [hunk 2]
```diff
@@ -79,7 +79,7 @@ Capabilities are evaluated as "out of the box" functionality. In cases where a s
 
 * _**Run Flash**:_ Can the software run Adobe Flash content?
 
-* _**Targeted scraping**:_ Can the user specify how subregions of web pages should be processed? (E.g., using some kind of rule-based scraping or pattern languages like XPath.)
+* _**Targeted scraping**:_ Can the user specify how subregions of web pages should be processed? (e.g., using some kind of rule-based scraping or pattern languages like XPath.)
```
Comment (mhucka) [inline web_archiving/readme.md:82]: (Same issue as the previous edit – see my comment above.)

**Review by mhucka — COMMENTED** [review]
I know @dcwalk already merged this, but I felt obliged to point out that there are minor issues with this PR.

File: web_archiving/readme.md (hunk 1 above)
Comment (dcwalk) [reply web_archiving/readme.md:52]: @mhucka --

I'm used to writing in APA and Chicago style where i.e., and e.g., are: 
- used in parenthesis
- undercase
- followed by a comma

I couldn't find an example in either where i.e. or e.g. was led by a capital ([chicago](http://www.chicagomanualofstyle.org/16/ch13/ch13_sec007.html), [apa](http://blog.apastyle.org/apastyle/2012/03/title-case-and-sentence-case-capitalization-in-apa-style.html)), and in both I believe lower case to be preferred. I'm happy to defer to another style if that is everyone's preference

Comment (mhucka) [reply web_archiving/readme.md:52]: (It is refreshing to be around people who actually care about details like this :-).)

You would indeed use _e.g._ and _i.e._ in lower case form when they introduce text that is part of another sentence. The examples given in that [APA blog posting](http://blog.apastyle.org/apastyle/2012/03/title-case-and-sentence-case-capitalization-in-apa-style.html) are of that kind. However, in the case of our README file, the parenthesized text is a full sentence—the previous sentence ends (with a question mark) and the parenthesized part is a new, separate sentence. Thus, as a separate sentence, the text should begin with a capital letter, even if that text is an abbreviation like e.g. or i.e. This is the difference between

> Was it a strange thing (i.e., was it was weird)?

and

> Was it a strange thing? (I.e., was it weird?)

Another way to see this is to replace i.e. with "that is" and then capitalize accordingly.

> Was it a strange thing (that is, was it was weird)?
> Was it a strange thing? (That is, was it weird?)

My very old paper copy of the APA manual doesn't touch on any of this, and I can't seem to access the Chicago manual online from my institution; the closest I can come to a rule right now is in the [Gregg Reference Manual](http://www.mhhe.com/business/buscom/gregg/toc.php) (I have the 10th edition in paper), where in rule #226, it is stated:

> If the item in parentheses is to be treated as a separate sentence,
> a) The preceding sentence should close with a punctuation mark of its own.
> b) The item in parentheses should begin with a capital letter.

(There's more to #226, but it's about the closing parenthesis and irrelevant to the issue at hand.) No exceptions are given for abbreviations.

The bottom line is that I don't think this is actually an issue of style preferences (and I'm certainly willing to use specific style guides if people prefer). It's about capitalizing the first word of a sentence.

(And if I'm wrong, I'll happily eat my words!)

Comment (mhucka) [reply web_archiving/readme.md:52]: Also, I don't mean to be argumentative.

**Comment by machawk1** [comment]
I have not encountered a sentence starting with I.e. or E.g. and am used to each term being using in parenthesis or following an em-dash within a sentence (akin to @dcwalk). I had not realized when I suggested the PR that each was its own sentence. I am open to either style so long as it's consistent. The primary point of the PR was to fix the missing paren.

**Comment by machawk1** [comment]
Encountered my first capitalized "E.g." in academic writing in a re-reading of Marshall and Shipman's, "[An Argument for Archiving Facebook as a Heterogeneous Personal Store](http://www.csdl.tamu.edu/~marshall/DL14-marshall-and-shipman.pdf)" from JCDL 2014 (§3.2) and was reminded of this PR/thread.

**Comment by ebarry** [comment]
shall we also debate "grammatic" vs "grammatical" ? ;)

**Comment by machawk1** [comment]
@ebarry I think that is unnecessary. I have never seen the former used. ;)

**Comment by mhucka** [comment]
Aside from providing an example of a sentence beginning with "E.g.," that paper provides some useful content and I'm really glad you mentioned it. I wasn't aware of it before.
